temporian/io/read_event.py

```diff
@@ -0,0 +1,40 @@
+from typing import List, Optional
+import pandas as pd
+from pathlib import Path
+from temporian.implementation.numpy.data.event import NumpyEvent
+
+
+def read_event(
+    path: str,
+    timestamp_column: str,
+    index_names: List[str] = [],
+    sep: str = ",",
+) -> NumpyEvent:
+    """Read a NumpyEvent from a file.
+
+    Args:
+        path (str): Path to the file.
+        timestamp_column (str): Name of the column to be used as timestamp for the event.
+        index_names (List[str], optional): Names of the DataFrame columns to be used as index for the event. Defaults to [].
+        sep (str, optional): Separator to use. Defaults to ",".
+
+
+    Returns:
+        NumpyEvent: NumpyEvent read from file.
+
+    Raises:
+        FileNotFoundError: If file does not exist.
+        ValueError: If file is not a csv file.
+
+    """
+    # check file exists and is a csv
+    path = Path(path)
+    if not path.exists():
+        raise FileNotFoundError(f"File {path} does not exist.")
+    if path.suffix != ".csv":
+        raise ValueError(f"File {path} is not a csv file.")
```

Comment:
pandas doesn't do this check, but I don't think we should either - the user might have valid csv content in a file marked with a .txt extension. I would just call pandas' method directly with whatever we receive as `path`.